spec/clientCalc.spec.mjs

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-undef */
+import { calculate } from '../src/client/utils/calculate.mjs';
+
+describe('addition', function() {
+    it('correctly adds', function() {
+        const answer = calculate("3 + 4 - 6");
```

Comment:
This test is doing more than a single _unit_ of logic (subtracting as well as adding), which usually means you have room to split this test into two tests

Reply:
Good point - I'd argue that _corectly adds_ is wrong, rather than the actual test, though - I'll pick up on that tomorrow - thanks!